tools/clang/test/HLSLFileCheck/validation/WaveSizeAttrAllowed5.hlsl

```diff
@@ -0,0 +1,8 @@
+// RUN: %dxc -T lib_6_7 %s 
+
+// This test just checks whether compilation succeeds
+// because we shouldn't presume that N() is an entry 
+// point, so entry point attributes are ignored.
```

Comment:
I think we at least need a CHECK-NOT for erros and warnings here. Since we're using the HLSL Filechecker, if dxc fails with an error, I don't think the test will fail.